.github/workflows/release-mockopampserver.yml

```diff
@@ -0,0 +1,94 @@
+# Release a tagged version of the '@elastic/mockopampserver' package.
+name: release-mockopampserver
+
+on:
+  push:
+    tags:
+      - mockopampserver-v*.*.*
+
+# 'id-token' perm needed for npm publishing with provenance (see
+# https://docs.npmjs.com/generating-provenance-statements#example-github-actions-workflow)
+permissions:
+  attestations: write
+  contents: write
+  packages: write
+  pull-requests: read
+  id-token: write
+
+jobs:
+  release:
+    runs-on: ubuntu-24.04
+    env:
+      PKGDIR: packages/mockopampserver
+      PKGNAME: "@elastic/mockopampserver"
+      DOCKER_IMAGE_NAME: ghcr.io/${{ github.repository }}/mockopampserver
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'v22.16.0'
+          registry-url: 'https://registry.npmjs.org'
+
+      # Push a Docker image.
+      - uses: docker/setup-buildx-action@v3
```

Comment:
for third party actions, for infosec reasons we recommend instead of specifying a tag `@v3` to instead specify a sha.